src/components/meetings/PersonMeeting.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useMemo } from 'react'
+
+import dayjs from 'dayjs'
+
+import { Avatar, Button, Chip, Paper, Typography } from '@mui/material'
+import { Box } from '@mui/system'
+
+export type Props = {
+  status: string
+  label: string
+  statusLabel: string
+  interviewer: string
+  interviewerId: string | number
+  interviewerName: string
+  interviewerAvatarUrl: string
+  interviewerLabel: string
+  intervieweeId: string | number
+  intervieweeName: string
+  intervieweeAvatarUrl: string
+  intervieweeLabel: string
+  startAt?: string
+  endAt?: string
+  dateLabel?: string
+  inviteUrl?: string
+  inviteUrlLabel?: string
+  userId: string | number
+  statusOptions: any[]
+}
+
+const PersonMeeting: React.FC<Props> = ({
+  status,
+  label,
+  statusLabel,
+  interviewerId,
+  interviewerName,
+  interviewerAvatarUrl,
+  interviewerLabel,
```

Comment:
você consegue colocar um valor default aqui `interviewerLabel = 'alguma coisa',`

Reply:
Coloquei os valores default, mas quando eu for utilizar esse componente, vou ter q passar de qualquer forma o valor do i18n